recipes/emsdk/all/conandata.yml

```diff
@@ -1,4 +1,19 @@
 sources:
+  "3.1.55":
```

Comment:
@johannes-wolf do you really need all these version? Otherwise, could please keep only the latest version available. Building all these versions will only consume CI resource basically. For who needs other versions, we can add by issue request.